test/common/secret/sds_api_test.cc

```diff
@@ -115,6 +115,92 @@ TEST_F(SdsApiTest, DynamicTlsCertificateUpdateSuccess) {
   handle->remove();
 }
 
+class PartialMockSds : public SdsApi {
+public:
+  PartialMockSds(NiceMock<Server::MockInstance>& server, Api::Api& api,
+                 NiceMock<Init::MockManager>& init_manager,
+                 envoy::api::v2::core::ConfigSource& config_source)
+      : SdsApi(
+            server.localInfo(), server.dispatcher(), server.random(), server.stats(),
+            server.clusterManager(), init_manager, config_source, "abc.com", []() {}, api) {}
+
+  MOCK_METHOD2(onConfigUpdate,
+               void(const Protobuf::RepeatedPtrField<ProtobufWkt::Any>&, const std::string&));
+  void setSecret(const envoy::api::v2::auth::Secret&) override {}
+  void validateConfig(const envoy::api::v2::auth::Secret&) override {}
+};
+
+// Basic test of delta's passthrough call to the state-of-the-world variant, to
+// increase coverage.
```

Comment:
It's a bit more than just coverage, since we have some new code and want to ensure it's doing the right thing. I think especially because this is otherwise unused by anyone today, it's valuable to have this.